stringcompare/distance/hamming.py

```diff
@@ -0,0 +1,22 @@
+#hamming distance
+from comparator import StringComparator
+
+def hamming(s, t):
+    if (len(s) != len(t)):
+        raise ValueError("Words must be of the same size")
+    distance = 0
+    for i in range(0, len(t)):
+        distance += (s[i] != t[i])
+    return distance
+
+class Hamming(StringComparator):
+    def __init__(self, similarity=False):
+        self.similarity = similarity
+
+    def compare(self, s, t):
+        l = len(s)
```

Comment:
`l` should be max(len(s), len(t)) if we allow comparing strings of different lengths. Also consider renaming to `ell` to avoid confusion with `1`.